src/utils/source-map.js

```diff
@@ -16,7 +16,7 @@ function restartWorker() {
     sourceMapWorker.terminate();
   }
   sourceMapWorker = new Worker(
-    getValue("baseWorkerURL") + "source-map-worker.js"
+    getValue("hostURL") + getValue("baseWorkerURL") + "source-map-worker.js"
```

Comment:
Can the config just duplicate the `hostURL` instead of adding more complexity here? I'd rather have absolute URLs in the config so you know exactly what URL it's using.